cypress/e2e/models/migration/reports/reports.ts

```diff
@@ -16,57 +16,34 @@ limitations under the License.
 /// <reference types="cypress" />
 /// <reference types="cypress-xpath" />
 
+import { selectItemsPerPage } from "../../../../utils/utils";
 import { applicationName, risk } from "../../../types/constants";
-import * as commonView from "../../../views/common.view";
-import { itemsPerPageMenu, itemsPerPageToggleButton } from "../../../views/reports.view";
+import {
+    articleCard,
+    articleExpandedContent,
+    articleHeader,
+    articleItem,
+    button,
+} from "../../../views/reports.view";
 
-export function selectItemsPerPageAdoptionCandidate(items: number): void {
-    cy.get(itemsPerPageMenu)
-        .find(itemsPerPageToggleButton)
-        .then(($toggleBtn) => {
-            if (!$toggleBtn.eq(0).is(":disabled")) {
-                $toggleBtn.eq(0).trigger("click");
-                cy.get(commonView.itemsPerPageMenuOptions);
-                cy.get(`li > button[data-action="per-page-${items}"]`).click();
-                cy.wait(2000);
-            }
-        });
-    cy.wait(2000);
-}
-
-export function selectItemsPerPageIdentifiedRisks(items: number): void {
-    cy.get(itemsPerPageMenu)
-        .find(itemsPerPageToggleButton)
-        .then(($toggleBtn) => {
-            if (!$toggleBtn.eq(1).is(":disabled")) {
-                $toggleBtn.eq(1).trigger("click");
-                cy.wait(1000);
-                cy.get(commonView.itemsPerPageMenuOptions);
-                cy.get(`li > button[data-action="per-page-${items}"]`).click();
-                cy.wait(2000);
-            }
+export function selectItemsPerPageinReports(items: number, articleTitle: string): void {
+    cy.log(`Select ${items} per page`);
```

Comment:
This log is not needed as there is already one in the `selectItemsPerPage` method